src/js/utils/Query.js

```diff
@@ -3,16 +3,15 @@
 import StringConvert from 'grommet/utils/StringConvert';
 
 const TRACE_PARSING = false;
-// don't convert timestamps, MAC addresses, or WWNs to attribute:value
-// This pattern matches the name: ^[^\d:'"\s]{2}[^:'"\s]+
+// This pattern matches the name: ^[^\d='"\s]{2}[^='"\s]+
 // We allow for the value to be optionally be quoted. So, we repeat the name
 // pattern three times, once for single quoted value, once for double quoted
 // value, and lastly with no quotes.
 // We don't build this programmatically for better performance.
 const ATTRIBUTE_PATTERN = new RegExp([
-  `^[^\d:'"\s]{1}[^:'"\s]*:'[^']+'`,
-  `^[^\d:'"\s]{1}[^:'"\s]*:"[^"]+"`,
-  `^[^\d:'"\s]{1}[^:'"\s]*:[^'"\s]+`].join('|'));
+  `^[^\d='"\s]{1}[^='"\s]*='[^']+'`,
```

Comment:
Interesting. I restored the original attribute:value parser. It's not a big deal for us anyway.